neasqc_wp61/models/quantum/embedder.py

```diff
@@ -21,29 +22,28 @@ class Embedder(ABC):
     comprising a dataset.
     """
 
-    def __init__(self, dataset: DataFrame, sentence_or_word: bool = True) -> None:
+    def __init__(self, dataset: DataFrame, sentence_embedding: bool = True) -> None:
```

Comment:
I feel this name should be different. If I just read this variable name, in this context, i assume this is the actual embedding, not a boolean.
So smth more like is_sentence_embedding or the like I would prefer.

Reply:
addressed

Reply:
No it isn't!